README.md

```diff
@@ -0,0 +1,98 @@
+#  Source Elements
+
+:bangbang: | This SDK is still in beta, and may change significantly before its public release.
+:---: | :---
+
+This project contains Source Elements, our React based member facing SDK. This SDK makes it easy for you to build member side experiences and uses an underlying `@source-health/client` that your provide to it to make calls using your API keys to the Source Health API. 
+
+## Installation
+
+### First, install `@source-health/elements-react`.
+
+Install with `yarn`:
+
+```
+yarn add @source-health/elements-react
+```
+
+OR with `npm`:
+
+```
+npm install --save @source-health/elements-react
+```
+
+##  Getting Started
+
+### Generating a Member JWT
+In order to use Source Elements you must generate a Member JWT using `@source-health/client` on your backend that gets passed to your application. This member JWT is generated on your backend using your private API keys and is what let's our API know you've authenticated your Member and that we can trust requests coming directly from your frontend on behalf of a given Member.
```

Comment:
Might be worth pointing out that you don't need to use source-health/client to generate a member token (not everyone will have a Node backend), but rather that the SDK can help you generate one.
let's -> lets
Lowercase member as well